trinity/nodes/full.py

```diff
@@ -23,6 +26,9 @@ def __init__(self, chain_config: ChainConfig) -> None:
         self._node_key = chain_config.nodekey
         self._node_port = chain_config.port
         self._max_peers = chain_config.max_peers
+        self._tx_pool = TxPool()
+        self.add_service(self._tx_pool)
```

Comment:
Shouldn't the light node also need to propagate the transaction pool? If so, this can go in `Node` instead of `FullNode`.

Reply:
That was indeed, one of the questions I had in mind. I wasn't sure if a light node relays transactions similar to a full node. Thanks for jumping in to clarify!